python/humbug/report.py

```diff
@@ -74,6 +92,23 @@ def __init__(
 
         self.is_excepthook_set = False
 
+    def create_report(
```

Comment:
I don't understand why you introduced `Reporter.create_report`. Older version of `humbug` uses `Reporter.publish`. We should stay consistent with that.

Reply:
I expected a reimplementation of `Reporter.publish` using this new `/humbug/reports` endpoint.

Reply:
done